app/models/slide.rb

```diff
@@ -87,6 +87,19 @@ def self.key_exist?(key)
     Slide.where('slides.object_key = ?', key).count > 0
   end
 
+  def self.update_after_convert(object_key, file_type, num_of_pages)
+    slide = Slide.where('slides.object_key = ?', object_key).first
```

Comment:
__[RuboCop]__ Rails/FindBy: Use `find_by` instead of `where.first`.

<sub>posted by Sider</sub>